src/main/java/com/ibr/fedora/testsuite/HttpPut.java

```diff
@@ -24,32 +24,54 @@
 import com.ibr.fedora.TestsLabels;
 import io.restassured.RestAssured;
 import io.restassured.config.LogConfig;
+
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Parameters;
 import org.testng.annotations.Test;
 
+import static org.hamcrest.CoreMatchers.containsString;
+
 import java.io.FileNotFoundException;
 import java.io.PrintStream;
 
 public class HttpPut {
+public String username;
+public String password;
 public TestsLabels tl = new TestsLabels();
+
 /**
+ * Authentication
+ * @param username
+ * @param password
+ */
+@BeforeClass
+@Parameters({"param2", "param3"})
+public void auth(final String username, final String password) {
+this.username = username;
+this.password = password;
+}
+
+/**
+ * 3.4-A
  * @param host
  */
-@Test(priority = 8)
+@Test(priority = 20)
 @Parameters({"param1"})
 public void httpPut(final String host) throws FileNotFoundException {
     final PrintStream ps = TestSuiteGlobals.logFile();
 
-    ps.append("\n8." + tl.httpPut()[0] + "-" + tl.httpPut()[1]).append("\n");
+    ps.append("\n20." + tl.httpPut()[0] + "-" + tl.httpPut()[1]).append("\n");
     ps.append("Request:\n");
     final String resource =
             RestAssured.given()
+.auth().basic(this.username, this.password)
```

Comment:
Ideally, we would not have to include `.auth()` in so many places. In other words, it would be good to refactor the request construction into a generalized method.
...but can be restructured later.